docs/setup/setup.md

```diff
@@ -17,110 +23,127 @@
     - --insecure-port=8080
     - --insecure-bind-address=0.0.0.0
     ```
```

Comment:
I think it's no need require user expose this insecure port, better to use https for default, just set the path-to-kubeconfig to cloudcore's config.

Reply:
ok